src/app/core/store/core/configuration/configuration.effects.ts

```diff
@@ -53,12 +52,10 @@ export class ConfigurationEffects {
         select(getCurrentLocale),
         mapToProperty('lang'),
         distinctUntilChanged(),
-        // https://github.com/ngx-translate/core/issues/1030
-        debounceTime(0),
         whenTruthy(),
         switchMap(lang => languageChanged$.pipe(mapTo(lang), take(1)))
       )
-      .subscribe((lang: string) => {
+      .subscribe(lang => {
```

Comment:
Where did the typing go? Is there a downside to it?

Reply:
It's no longer necessary because of the `mapTo` in the previous line.